resumegenerator.html

```diff
@@ -666,32 +671,56 @@ <h5 class="text-center my-2">A rough preview. Differs from the LaTeX output. <br
     </pre>
 
     <!-- Latex code ends-->
+  <!-- updating footer -->
 
-    <footer class="footer">
-        <div class="container">
-            <div class="footer-content">
-                <div class="footer-logo">
-                    IIT Gandhinagar
-                </div>
-                <div class="footer-links">
-                    <ul class="footer-menu">
-                        <li><u><a href="https://github.com/Kishan-Ved/resume_generator" target="_blank">Github</a></u>
-                        </li>
-                    </ul>
-                </div>
-                <div class="footer-social">
-                    <ul class="social-icons">
-                        <li>kishan.ved@iitgn.ac.in</li><br>
-                    </ul>
-                </div>
+  <footer class="footer">
+    <div class="container">
+        <div class="footer-content">
+            <div class="footer-logo">
+                IIT Gandhinagar
             </div>
-            <div class="footer-bottom">
-                <p>To report a bug or add a new feature, use the GitHub link above and make a PR. <br> Made with
-                    &hearts; by <u><a href="https://github.com/Kishan-Ved" style="color: white;" target="_blank">Kishan
-                            Ved</a></u>.</p>
+            <div class="footer-links">
+                <ul class="footer-menu">
+                    <!-- <li>
+                        <u>
+                            <a href="https://github.com/Kishan-Ved/resume_generator" target="_blank">
+                                <i class="fab fa-github"></i> Github
+                            </a>
+                        </u>
+                    </li> -->
+                </ul>
             </div>
+            <div class="footer-social">
+                <ul class="social-icons">
+                    <li>
+                        <i class="fas fa-envelope"></i> kishan.ved@iitgn.ac.in
+                    </li>
+                </ul>
+            </div>
+        </div>
+        <div class="footer-follow">
+            <p>Follow Us:</p>
+            <ul class="follow-icons">
+                <li>
+                    <a href="https://github.com/Kishan-Ved" target="_blank">
```

Comment:
Please replace this link by https://github.com/Kishan-Ved/resume_generator

Reply:
Sure